discord/interactions/index.ts

```diff
@@ -209,7 +217,9 @@ export const setupInteractions = (client: Client) => {
         case "chatInput":
           return (
             interaction.isChatInputCommand() &&
-            interaction.commandName === config.identifier
+            (config.matchType === "startsWith"
```

Comment:
Cancel this change. Slash commands are static, so they always exactly match the string